gtsam/sfm/MFAS.h

```diff
@@ -0,0 +1,108 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010-2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+#ifndef __MFAS_H__
+#define __MFAS_H__
+
+#include <gtsam/geometry/Unit3.h>
+#include <gtsam/inference/Key.h>
+
+#include <map>
+#include <memory>
+#include <vector>
+
+namespace gtsam {
+
+// used to represent edges between two nodes in the graph
+using KeyPair = std::pair<Key, Key>;
+using TranslationEdges = std::map<KeyPair, Unit3>;
```

Comment:
maybe name it UnitTranslationEdges?

Reply:
While I do appreciate the name, I used the same that was used in TranslationRecovery.h for consistency. We might not use TranslationEdges now that we are changing the interface to vector of edges, so I'm not changing the name right now.